dts/bindings/radio_coex/nordic,nrf700x-coex.yaml

```diff
@@ -0,0 +1,37 @@
+# Copyright (c) 2022 Nordic Semiconductor ASA
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+
+description: |
+    This is a representation of an external radio coexistence setup that has a
+    three-pin control interface (REQUEST, GRANT, PRIORITY/DIRECTION).
+
+compatible: "nordic,nrf700x-coex"
+
+include: base.yaml
+
+properties:
+    req-gpios:
+        type: phandle-array
+        required: true
+        description: |
+            GPIO of the SOC connected to the PTA's REQUEST pin.
+
+    pri-dir-gpios:
+        type: phandle-array
+        required: true
+        description: |
+            GPIO of the SOC connected to the PTA's PRIORITY pin.
+            This GPIO is also used to indicate direction (TX/RX).
```

Comment:
The pin name COEX_STATUS0 (instead of PRIORITY) is used in Internal Product Specification.
We should use STATUS0 instead of PRIORITY which is misleading (clash with desctiprion for swctrl1-gpios see ...SOC controlling the Priority ).

I'm not sure if `pri-dir-gpios` should also be reverted to coex-status0-gpios as it used to be.

@krish2718 , what's your opinion?

Reply:
I think that indeed `pri-dir-gpios` also should be changed to `status0-gpios`

Reply:
Ok, fixed